terraform/azure-cosmosdb/provision-cosmosdb-sql.tf

```diff
@@ -119,11 +119,11 @@ resource "azurerm_cosmosdb_sql_database" "db" {
 
 output "cosmosdb_host_endpoint" { value = azurerm_cosmosdb_account.cosmosdb-account.endpoint }
 output "cosmosdb_master_key" {
-  value     = azurerm_cosmosdb_account.cosmosdb-account.primary_master_key
+  value     = azurerm_cosmosdb_account.cosmosdb-account.primary_key
```

Comment:
Nice change - I think the right thing in the long term is to deprecate our use of `cosmosdb_master_key` and `cosmosdb_readonly_master_key` in favor of `cosmosdb_readonly_key` and `cosmosdb_key` as well but not needed in this PR